backends/advanced/src/advanced_omi_backend/routers/modules/queue_routes.py

```
@@ -408,25 +435,28 @@ async def clear_events(
         raise HTTPException(status_code=500, detail=f"Failed to clear events: {str(e)}")
 
 
-
 @router.get("/stats")
-async def get_queue_stats_endpoint(
-    current_user: User = Depends(current_active_user)
-):
+async def get_queue_stats_endpoint(current_user: User = Depends(current_active_user)):
```

Ruff (0.15.1)

[warning] Unused function argument: `current_user`

(ARG001)